pd-client/client_test.go

```diff
@@ -175,6 +176,25 @@ func (s *testClientSuite) TestTSO(c *C) {
 	}
 }
 
+func (s *testClientSuite) TestTSORace(c *C) {
+	var wg sync.WaitGroup
+	begin := make(chan struct{})
+	count := 10
+	wg.Add(count)
+	for i := 0; i < count; i++ {
+		go func(begin chan struct{}, wg *sync.WaitGroup) {
```

Comment:
No difference. `go func() {}` is closure, closure is slightly poor in performance.
I'll update it here, for readability.